demo/java/src/ai/picovoice/rhinodemo/FileDemo.java

```diff
@@ -215,6 +228,10 @@ private static Options BuildCommandLineOptions() {
                         "fewer misses at the cost of (potentially) increasing the erroneous inference rate. " +
                         "If not set 0.5 will be used.")
                 .build());
+
+        options.addOption(new Option("e", "require_endpoint", false, "If set to `False`, Rhino does not require an " +
```

Comment:
Since false is the default and this cmd option is a flag, we should just say "If set, Rhino requires a ..."